drivers/usb/udc/udc_dwc2.c

```diff
@@ -674,22 +674,35 @@ static void dwc2_handle_enumdone(const struct device *dev)
 	priv->enumspd = usb_dwc2_get_dsts_enumspd(dsts);
 }
 
-static inline int dwc2_read_fifo_setup(const struct device *dev)
+static inline int dwc2_read_fifo_setup(const struct device *dev, uint8_t ep,
+				       const size_t bytes)
```

Comment:
Please change to `const size_t size` to be more consistent with e.g. dwc2_read_fifo().